### PR TITLE
Minor logger connector cleanup [1/n]

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

Part of #7183

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [n/a] Did you make sure to update the documentation with your changes? (if necessary)
- [n/a] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [n/a] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

### CODE REVIEW

File: pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py [hunk 1]
```diff
@@ -474,7 +469,7 @@ def __call__(
             batch_idx: Batch index seen during batch training or evaluation.
                 Works only with ``reduced=False``
 
-            split_idx: Index of split idx in training loop when ttbt is used.
+            split_idx: Index of split idx in training loop when ttbpt is used.
```
Comment (awaelchli) [inline pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py:472]: tbptt

File: pytorch_lightning/core/lightning.py [hunk 2]
```diff
@@ -310,55 +310,49 @@ def log(
...
+                on_epoch=on_epoch,
+                reduce_fx=reduce_fx,
+                tbptt_reduce_fx=tbptt_reduce_fx,
+                tbptt_pad_token=tbptt_pad_token,
+                enable_graph=enable_graph,
+                sync_dist=sync_dist,
+                sync_dist_op=sync_dist_op,
+                sync_dist_group=sync_dist_group,
+                sync_fn=self.trainer.training_type_plugin.reduce,
+                dataloader_idx=self._current_dataloader_idx if add_dataloader_idx else None,
```
Comment (awaelchli) [inline pytorch_lightning/core/lightning.py:354]:
```suggestion
                dataloader_idx=(self._current_dataloader_idx if add_dataloader_idx else None),
```
:)

**Review by awaelchli — APPROVED** [review]
good stuff! let's get it!

**Review by tchaton — APPROVED** [review]
LGMT !

**Review by ananthsub — APPROVED** [review]

**Pull request merged**